newsplease/examples/commoncrawl.py

```diff
@@ -34,30 +36,31 @@
 
 ############ YOUR CONFIG ############
 # download dir for warc files
-my_local_download_dir_warc = './cc_download_warc/'
+my_local_download_dir_warc = './cc_download_warc_test/'
```

Comment:
Could you please revert the changes in the config section back to the original content?